modules/utxo-lib/src/bitgo/UtxoPsbt.ts

```diff
@@ -992,52 +992,80 @@ export class UtxoPsbt<Tx extends UtxoTransaction<bigint> = UtxoTransaction<bigin
     return { tapOutputKey, participantPubKey, pubNonce: Buffer.from(pubNonce) };
   }
 
-  private setMusig2NoncesInner(keyPair: BIP32Interface, keyType: 'root' | 'derived', sessionId?: Buffer): this {
+  private setMusig2NoncesInner(
+    keyPair: BIP32Interface,
+    keyType: 'root' | 'derived',
+    sessionId?: Buffer,
+    inputIndex?: number
```

Comment:
if you change this to `inputs = this.data.inputs` and pass `[this.data.inputs[inputIndex]]` at the callsite, it is a bit safer because the `checkForInput()` calls are there

Reply:
Agreed. It's merged already. We will do it the recommended change later. I guess, for now it's safer, since it is private method.